### PR TITLE
doc: stylish fsm.md, #24717

### PR DESCRIPTION
**Pull request by patriknw**
Refs #24717

### CODE REVIEW

**Comment by patriknw** [comment]
PLS BUILD

File: akka-actor-typed-tests/src/test/java/jdocs/akka/typed/FSMDocTest.java [hunk 1]
```diff
@@ -14,182 +14,176 @@
...
+
+      public SetTarget(ActorRef<Batch> ref) {
+        this.ref = ref;
+      }
 
-    public ActorRef<Batch> getRef() {
-      return ref;
+      public ActorRef<Batch> getRef() {
+        return ref;
+      }
```
Comment (johanandren) [inline akka-actor-typed-tests/src/test/java/jdocs/akka/typed/FSMDocTest.java]: Do we really go with getters on these over just making the final message fields public?

**Review by johanandren — APPROVED** [review]
LGTM

File: akka-actor-typed-tests/src/test/java/jdocs/akka/typed/FSMDocTest.java (hunk 1 above)
Comment (patriknw) [reply akka-actor-typed-tests/src/test/java/jdocs/akka/typed/FSMDocTest.java]: we typically use `public final` field, I can change this as well

Comment (johanandren) [reply akka-actor-typed-tests/src/test/java/jdocs/akka/typed/FSMDocTest.java]: Good with less verbosity in the samples I think, makes them easier to digest quickly.

**Pull request merged**